build_tools/requirements_conda.txt

```diff
@@ -7,3 +7,4 @@ nltk==3.2.1
 pytest==2.9.1       
 flask==0.12         # server
 requests==2.11.1
+matplotlib==1.5.3
```

Comment:
here is what i get after making rebase, when trying to run `conda_setup.bat`:
"""
PackageNotFoundError: Package not found: '' Packages missing in current win-64 channels:
  - requests ==2.13.0
  - marshmallow ==2.12.2
  - webargs ==1.5.3
  - flask-apispec ==0.3.2

You can search for packages on anaconda.org with

    anaconda search -t conda webargs

(and similarly for the other packages)
"""

When i change `requests` version back to `2.11.1` (the last one used in `requirements_conda.txt` before rebase), this package disappears from the list of 4 ones causing problem.

Concerning `marshmallow` and `webargs`, even when i put their versions back to `2.10.2` and `1.4.0` respectively, this error persists, as far as for `flask_apispec`. Maybe the reason is that these 3 packages were being installed by `pip` (they were in `requirements-pip.txt`), and not by Conda. Apparently, it works ok in unix (otherwise you would change the way you package), but not on my Windows PC.